source/collaborate/channel-types.rst

```diff
@@ -29,13 +29,17 @@ Channel members can choose to leave private channels at any time.
 Direct messages
 ---------------
 
-Direct messages are channels for conversations between two people. You can also start a direct message with people on other teams when enabled by the System Admin. Only members of the conversation can see direct messages and channel heading information, including last active status of the other user, when enabled.
+Direct messages are channels for conversations between 2 people. You can start a direct message with people on other teams `when enabled by the system admin </configure/site-configuration-settings.html#users-restrictdirectmessage>`__. Only members of the conversation can see direct messages and channel heading information, including the last active status of the other user. 
+
+Direct messages always display a new message badge. Direct messages always trigger a notification, unless you mute the channel, or you disable notifications altogether. See the `notification documentation </preferences/manage-your-notifications.html>`__ for details on customizing notifications based on your preferences.
```

Comment:
Propose rewording to "Direct messages increments the numbered badge. It will trigger a notification unless the direct message is muted or your notifications is disabled altogether..."